backend-tests/tests/test_artifact.py

```diff
@@ -549,7 +549,7 @@ def test_artifacts_exclusive_to_user(self, mongo, clean_mongo):
 
             # create and upload the mender artifact
             with get_mender_artifact(
-                artifact_name=user.name,
+                artifact_name=user.name.translate({ord(c): None for c in ".@+"}),
```

Comment:
You could store this value in a variable and reuse it.

Reply:
done